### PR TITLE
 migrate to helmv3 client

### PR DESCRIPTION
**Pull request by kaladaOpuiyo**
This removes all dependencies on helm v2 (and tiller) and migrates to utilizing the helm v3 action clients. We also move away from vendoring in favor of go modules.

### CODE REVIEW

File: cmd/export.go [hunk 1]
```diff
@@ -12,15 +12,18 @@ import (
...
+var deployed bool
+var failed bool
 var mgrstate *state.State
 var pollingInterval int
 
 var exportCmd = &cobra.Command{
 	Use:   "export",
 	Short: "Export Helm release state",
 	Long: `
-Release Manager Export will contact Tiller in the configured cluster, collect
```
Comment (woz5999) [inline cmd/export.go:23]: looks weird. should re-wrap all the text to 80 chars

File: Dockerfile [hunk 2]
```diff
@@ -15,13 +15,12 @@ RUN chmod +x ./scripts/test.sh; sync; ./scripts/test.sh
...
 RUN apk --update add ca-certificates \
     && rm -rf /var/cache/apk/* \
     && rm -rf /var/lib/apk/*
 WORKDIR /app
 COPY --from=build /releasemanager /bin
 COPY --from=test /coverage.txt /coverage.txt
 
-ENTRYPOINT ["releasemanager"]
-CMD ["watch"]
+ENTRYPOINT ["releasemanager"]
```
Comment (woz5999) [inline Dockerfile:26]: why no newline?

File: pkg/export/stats.go [hunk 3]
```diff
@@ -1,4 +1,4 @@
-package export
+package export
```
Comment (woz5999) [inline pkg/export/stats.go:1]: cruft

File: Dockerfile (hunk 2 above)
Comment (kaladaOpuiyo) [reply Dockerfile:26]: Slipped

**Pull request merged**